src/Creitive/Breadcrumbs/Breadcrumbs.php

```diff
@@ -411,11 +412,15 @@ protected function renderCrumb($name, $href, $isLast = false)
 
 		if (!$isLast)
 		{
-			return "<li><a href=\"{$href}\">{$name}</a>{$divider}</li>";
+			return '<li itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem" >'
+				. "<a itemprop=\"item\" href=\"{$href}\"><span itemprop=\"name\">{$name}</span></a>"
+				. "<meta itemprop=\"position\" content=\"{$position}\" />{$divider}</li>";
 		}
 		else
 		{
-			return "<li class=\"active\">{$name}</li>";
+			return '<li itemprop="itemListElement" itemscope itemtype="http://schema.org/ListItem" '
+				. "class=\"active\"><span itemprop=\"name\">{$name}</span>"
+				. "<meta itemprop=\"position\" content=\"{$position}\" /></li>";
```

Comment:
Same comment about `$position` as above.

Honestly, I don't personally like this much HTML in strings, but since this package doesn't use any kind of views/templates, I guess we're stuck with that.